ra_aid/llm.py

```
@@ -1,93 +1,215 @@
 import os
-from langchain_openai import ChatOpenAI
+from typing import Any, Dict, Optional
+
+known_temp_providers = {"openai", "anthropic", "openrouter", "openai-compatible", "gemini", "deepseek"}
+
+from .models_params import models_params
```

Ruff (0.8.2)

Module level import not at top of file

(E402)